src/Spread/AdminSpread.php

```diff
@@ -13,11 +13,11 @@
 
 namespace Sonata\TimelineBundle\Spread;
 
+use Doctrine\Common\Persistence\ManagerRegistry;
```

Comment:
We need `doctrine/persistence` as a dependency

Reply:
Indeed, it makes more sense. I went straight to v1.3 of doctrine/persistence and use the non deprecated class Doctrine\Persistence\ManagerRegistry instead.